modules/evvtgendoc/compile.php

```diff
@@ -678,11 +744,18 @@ function eval_paracada($condition, $id, $module, $check = false) {
 		$values = explode(',', $valstr);
 	}
 
+	preg_match('/(\w+)\s\((.+)+\)/', $condition, $cond_elements); // Multiple conditions?
+	if (count($cond_elements) > 0 && isset($cond_elements[2])) {
+		$json_condition = make_json_condition($cond_elements[1], $cond_elements[2]);
+		$comp = 'wfeval';
+	}
+
 	for ($i=0; $i<count($condition_pair); $i++) {
 		$condition_pair[$i] = trim($condition_pair[$i]);
 	}
 
 	$token_pair = explode('.', $condition_pair[0]);
+	$token_pair[0] = explode(' ', $token_pair[0])[0];
```

Comment:
this looks dangerous, I understand that you are separating the conditions and it will do nothing if they are not there but I would prefer that you put this inside the "Multiple Conditions" check you do above, just to be safe.
The same comment about array access that I indicated above applies here too.

Reply:
It's been fixed. If I remember correctly the problem here was that when you use multiple conditions, a space will remain in the module name. Normally the modulename is immediately followed by a period/dot, but in the multiple conditions syntax there will most likely be an unwanted space that will prevent code further down to use the module name